packages/flet/lib/src/controls/elevated_button.dart

```diff
@@ -147,7 +151,7 @@ class _ElevatedButtonControlState extends State<ElevatedButtonControl>
               description:
                   "\"icon\" must be specified together with \"text\".");
         }
-        button = ElevatedButton.icon(
+        elevatedButton = ElevatedButton.icon(
```

Comment:
I think we can have the condition somewhere here, to avoid building buttons which wont be used.

Also, instead of three variables for each button we could just have one variable which will then be set as we check the type of button added to the UI.

Reply:
Updated to check condition and create different buttons in two places - if icon is not empty/empty